easybuild/easyconfigs/a/AlphaFold/AlphaFold-2.2.0-foss-2021a-CUDA-11.3.1.eb

```diff
@@ -0,0 +1,198 @@
+easyblock = 'PythonBundle'
+
+name = 'AlphaFold'
+version = '2.2.0'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://deepmind.com/research/case-studies/alphafold'
+description = "AlphaFold can predict protein structures with atomic accuracy even where no similar structure is known"
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+builddependencies = [
+    # CMake/Doxygen/SWIG are required for building OpenMM
+    ('CMake', '3.20.1'),
+    ('Doxygen', '1.9.1'),
+    # required for installing dm-tree
+    ('Bazel', '3.7.2'),
+    ('SWIG', '4.0.2'),
+]
+
+dependencies = [
+    ('Python', '3.9.5'),
+    ('CUDA', '11.3.1', '', True),
+    ('SciPy-bundle', '2021.05'),
+    ('PyYAML', '5.4.1'),
+    ('TensorFlow', '2.6.0', versionsuffix),
+    ('Biopython', '1.79'),
+    ('HH-suite', '3.3.0'),
+    ('HMMER', '3.3.2'),
+    ('Kalign', '3.3.1'),
+    ('jax', '0.2.24', versionsuffix),  # also provides absl-py
```

Comment:
`axis_size` was added to `jax.vmap` in https://github.com/google/jax/commit/50e7e952bde9a4a3ef9151b7b6d96803bc6796ea and that is tagged 0.2.26. We are using jax 0.2.24 as a dependency here, so an newer version is necessary.

Reply:
fixed, now using jax 0.3.9